lte/gateway/python/magma/pipelined/tests/test_ifaces.py

```diff
@@ -0,0 +1,32 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+from unittest.mock import patch
+
+import pytest
+from magma.pipelined.ifaces import get_mac_address
+
+
+@patch("magma.pipelined.ifaces.netifaces")
+def test_get_loopback_mac_address(netifaces_mock):
+    netifaces_mock.AF_LINK = 13
+    netifaces_mock.ifaddresses.return_value = {netifaces_mock.AF_LINK: [{"addr": "1.2.3.4"}]}
```

Comment:
nit: it's a mock and we can write what we want, but is it a bit confusing to mock to a ip-ish value? Would a mac-ish value (e.g., 11:22:33:44:55:66) be more appropriate?